codegen/templates/tchannel_endpoint.tmpl

```diff
@@ -45,6 +46,14 @@ func (h *{{$handlerName}}) Register(g *zanzibar.Gateway) error {
 		"{{.ThriftService}}",
 		"{{.Name}}",
 		h,
+		g.AllHostScope.Tagged(map[string]string{
```

Comment:
Instead of passing around the scope, it's better to define scope on the [handler](https://github.com/uber/zanzibar/pull/199/files#diff-7c8f7ec39bf077182fa0b404b3d49d76R38) and initialize it when a new handler is created.

Reply:
Yes, will do.